zospy/analyses/base.py

```diff
@@ -2,6 +2,18 @@
 from collections.abc import MutableMapping
 
 
+def process_oncomplete(analysis, oncomplete, ret):
```

Comment:
This functionality is already part of the `Analysis` class

Reply:
Yes. But sorry, I am not sure how I can remove this file change from the pull request. Can you do it somehow?